tests/scale_test/subscription_queries/smartseq2_query.json

```diff
@@ -0,0 +1,785 @@
+{
```

Comment:
These files are getting pretty large. Is there a to programatically generate this JSON that would take less lines of code?

Reply:
Yes, there's a script in Lira that generates this!

Reply:
Unfortunately the queries are duplicated here...